Makefile

```diff
@@ -20,12 +20,21 @@ config.mk:
 	exit 1
 
 setup.bin: setup.ml
-	@ocamlopt.opt -o $@ $< || ocamlopt -o $@ $< || ocamlc -o $@ $<
-	@rm -f setup.cmx setup.cmi setup.o setup.cmo
+	@ocamlfind ocamlopt -o $@ -linkpkg -package oasis.dynrun setup.ml || ocamlfind ocamlc -o $@ -linkpkg -package oasis.dynrun setup.ml || true
```

Comment:
Did we deliberately add the `|| true` to the end, does that mean that this target can succeed even if all the compilation commands fail, and we won't have the `setup.bin` that we need for later stages?